packages/components/switch/src/Switch.stories.tsx

```diff
@@ -28,6 +29,18 @@ export const Default: StoryFn = _args => (
   </div>
 )
 
+export const ImprovedField: StoryFn = _args => (
```

Comment:
I think that would be useful to have the following 4 examples:
- Invalid switch
- Switch with helper message
- Switch as required

Remember that if you add the `FormField.Label` the switch will have 2 labels

Reply:
I'm not so sure, for the same reasons I'm not convinced we should add tests about FormField+Switch combo here. Those are not directly related to the component, but to an implementation.

Reply:
I agree with @soykje , it's a "separation of concern" matter. 

I am already 50/50 about each component having a story to show how to combine it with `FormField`. So having every possible example I think will be too much to maintain whenever we update `FormField`. 

Regarding the "2 labels" thing maybe it can be an issue I am not sure, when I inspect the story it's true that the accessible name of the switch is no longer its label, but the formField label that replaces it. But maybe it's ok.
<img width="1197" alt="Capture d’écran 2023-05-25 à 15 29 40" src="https://github.com/adevinta/spark/assets/2033710/de047ca3-d31c-42c7-a1f7-21ca791955fe">